client/packages/system/src/Location/Components/LocationSearchInput.tsx

```diff
@@ -8,16 +14,52 @@ interface LocationSearchInputProps {
   onChange: (location: LocationRowFragment | null) => void;
   disabled: boolean;
   autoFocus?: boolean;
+  canRemove?: boolean;
 }
 
+interface LocationOption {
+  label: string;
+  value: string | null;
+}
+
+const optionRenderer = (
+  props: React.HTMLAttributes<HTMLLIElement>,
+  location: LocationOption
+) => {
+  const { style, ...rest } = props;
+
+  return location.value === null ? (
+    <MenuItem
+      {...rest}
+      sx={{
+        ...style,
+        display: 'inline-flex',
+        flex: 1,
+        width: '100%',
+        borderTop: '1px solid',
+        borderTopColor: 'divider',
+      }}
+      key={location.label}
+    >
+      <span style={{ whiteSpace: 'nowrap', flex: 1 }}>{location.label}</span>
+      <CloseIcon sx={{ color: 'gray.dark' }} />
+    </MenuItem>
+  ) : (
+    <MenuItem {...props} key={location.label}>
+      <span style={{ whiteSpace: 'nowrap' }}>{location.label}</span>
+    </MenuItem>
+  );
+};
+
 export const LocationSearchInput: FC<LocationSearchInputProps> = ({
   selectedLocation,
   width,
   onChange,
   disabled,
   autoFocus = false,
+  canRemove = false,
```

Comment:
I thought we wanted to allow location removal everywhere applicable 🤔

Reply:
it would be good - though I tested and it doesn't work for Stock Lines. you remove, save and the value pops back in again. So I thought that until we have api support, we should disable the f/e option. I'll make another issue!